src/paperqa/agents/tools.py

```diff
@@ -309,9 +309,6 @@ async def gen_answer(self, state: EnvironmentState) -> str:
         Args:
             state: Current state.
         """
-        if not state.docs.docs:
-            raise EmptyDocsError("Not generating an answer due to having no papers.")
```

Comment:
Note that `state.session.raw_answer` was left untouched here, that is a bug this PR fixes